src/convertToObject.js

```diff
@@ -12,7 +12,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const styleObject = {};
```

Comment:
Уникай у назвах змінних слів `object`, `array`, `list`
```suggestion
  const styles = {};
```